### DISCUSSION TITLE
How to render smoothed line series?

### DISCUSSION THREAD

**Discussion by andersonf78** [tradingview/lightweight-charts#641] [Q&A] [answered]
Hello all,

I'm working on a chart containing a candlestick series along with a line series (for moving average).

It works fine, but the the MA renders as straight lines (as expected). 

Is there an option to get a line series to render with smoothing?

Thanks!

**Comment by timocov** [comment, marked as answer]
Is #506 what you're looking for?

**Reply by andersonf78** [reply]
Exactly that! Thanks for the quick answer! It's good to know it's on the radar!
